scripts/adjustEnvVars.sh

```diff
@@ -6,6 +6,20 @@ WHT="\e[37m"
 CLR="\e[0m"
 NWL="\n"
 
+ORIGINAL_CFLAGS="$CFLAGS"
+ORIGINAL_CXXFLAGS="$CXXFLAGS"
+
+# Platform dependent commands
+if [[ $(uname -s) = "Linux" ]]; then
```

Comment:
Use a `case` statement and check for unknown platforms, exiting with an error for unknown ones. There are other similar examples on the file.